README.md

```diff
@@ -5,7 +5,37 @@ You might not care when launching a single container on your laptop, but in the
 
 ## The problem
 
-TBD.
+### Too many containers require privileges to run.
```

Comment:
It seems strange to not mention e.g. [Kubernetes MustRunAsRange](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/security-context-constraints.md) here.